### ISSUE TITLE
data.table error/warning when using read_vhdr

### ISSUE THREAD

**Issue by stonekate** [bnicenboim/eeguana#169]
For recordings exported from the second lab laptop, I get the following errors/warnings when using read_vhdr:

![image](https://user-images.githubusercontent.com/12735809/137095558-6f02a2c7-f96b-4841-966d-4b68346a68da.png)

It's specific to that laptop! Exported data from the main laptop can be imported fine. In case you forgot, the process is: record in .cnt format, export to .eeg format (Brainvision), then import into R with eeguana. I will put 1 example in Dropbox now.

**Comment by bnicenboim** [comment]
Hi, I just got to this. 
I don't think the error is eeguana's side. The binary file has the wrong size. Maybe it didn't finish creating the ".eeg" file? Have you tried to export them again?

**Comment by stonekate** [comment]
No worries... yeah I tried re-exporting. Weird! I'll look into it.

**Comment by bnicenboim** [comment]
could you open those files with Brainvision?

**Comment by stonekate** [comment]
I can't remember, I'll try it next week

**Comment by stonekate** [comment]
(sorry for delayed response) I now checked, they all opened fine in BrainVision Analyzer. So I just exported them as binary .dat files from there and can now import them into eeguana without any problems. So from my side it's resolved :)

**Comment by bnicenboim** [comment]
wait, so the same files that you can't open with eeguana can be opened with Brainvision? That's strange.
Can you send me the two versions of the file to see how BV deals with the incomplete rows?

**Comment by stonekate** [comment]
oh man, sorry, I wrote down and exported the wrong bloody file numbers.... ignore everything I just said, I'll try again next week.

**Comment by stonekate** [comment]
Ok now I checked the right files: BrainVision could load and export them in a way that eeguana could then load, but either I don't think BV dealt with the issue properly or there is something very wrong with the data (probably the latter). The image attached is the data loaded and filtered in eeguana, but this is also what it looked like in BV. It also looked weird in asalab once I reopened it, but I assume it didn't look like that while being recorded. I've put some files in Dropbox (40 regular and BV export) in case you want to look at why BV could load it but eeguana couldn't, but I think the core problem is something has gone wrong with asalab. Which is not your problem :)

![image](https://user-images.githubusercontent.com/12735809/152825507-ac902a4d-8a93-42f9-9499-bcb87a988a70.png)
